Analysis/tfcrig/analysis.py

```diff
@@ -525,7 +546,7 @@ def get_data_features_from_data_file(
             "z_learning_rate": z_learning_rate,
             "z_learning_rate_reward": z_learning_rate_reward,
         }
-        if dict_contains_other_values(features_dict, (np.generic, str)):
+        if dict_contains_other_values(features_dict, (np.generic, str, float)):
```

Comment:
This update was necessary because sometimes lick frequency is `nan` -- it seems like it shouldn't be assuming that every session contains CS+, CS- trial types but perhaps that is not true

Reply:
This makes sense, there are some trials with just one trial type (only CS-) for conditioning